### ISSUE TITLE
UploadedItem view doesn't have a max width

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/dpul#1114]
This results in a giant banner in pursuit of the picturesque: 

https://dpul-staging.princeton.edu/picturesque

![Screen Shot 2021-10-13 at 12.19.31 PM.png](https://images.zenhubusercontent.com/5c8fd280aa8dca0a81398108/453661cd-ff4b-40fb-83f5-e22b0d2bdb9a)

**Comment by kelea99** [comment]
yup :) it was a way to get them large enough for folks to view, in the order they want, while still linking them to the viewer or to the lending institution. It worked well
